behave/features/steps/common_steps.py

```diff
@@ -276,3 +279,29 @@ def step_impl_create_case(context):
     context.case_reference = context.helperfunc.find_by_css_selector(
         "h1.CaseBar-caseNum a"
     ).text
+
+
+def get_tag(context, find_tag):
+    return [tag for tag in context if find_tag in tag]
+
+
+def make_dir(dir):
+    """
+    Checks if directory exists, if not make a directory, given the directory path
+    :param: <string>dir: Full path of directory to create
+    """
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+
+
+def check_accessibility(context):
+    # Sleep prevents Axe exceptions.
+    # If no logs for Axe, Axe is called too fast when trying to inject javascript.
+    time.sleep(0.6)
+    axe = Axe(context.helperfunc.driver())
+    axe.inject()
+    results = axe.run()
+    if len(results["violations"]) > 0:
+        make_dir("feature_errors")
+        axe.write_results(results, "feature_errors/ally.json")
+    return len(results["violations"]) == 0, axe.report(results["violations"])
```

Comment:
why are you returning two values from the function when you only use the first?

Reply:
Ah great catch! I thought that was returning a boolean but it's not. It returns a boolean for `len(results["violations"]) == 0` but not for `axe.report(results["violations"]` which is actually the error messages.